src/frame/endpoints/UserApiEndpoints.ts

```diff
@@ -78,12 +78,13 @@ function verify(): RequestMeta {
  * @param  {UserKeys} keys Generated keys. Can either be a full key pair set of public/private keys, or a partial set with only a public key.
  * @return {RequestMeta}                Request object that can be passed to fetch() API
  */
-const userCreate = (keys: UserKeys): RequestMeta => {
+const userCreate = (keys: UserKeys, needsRotation: boolean): RequestMeta => {
     const {publicKey, encryptedPrivateKey} = keys;
 
     const body = {
         userPublicKey: publicKeyToBase64(publicKey),
         userPrivateKey: fromByteArray(encryptedPrivateKey),
+        needsRotation: needsRotation,
```

Comment:
If the key and the variable both have the same value, you can shorthand this like

```suggestion
        needsRotation,
```